platform-validation/scripts/compatibility.sh

```diff
@@ -12,6 +12,30 @@ ARCH=""
 #  Text Formatting
 #  Derived from : https://github.com/moby/moby/blob/master/contrib/check-config.sh
 # ------------------------------------------------------------------------------
+
+possibleConfigs="
```

Comment:
variables names are all have to be uppercase. can you change it to `POSSIBLE_CONFIGS`?